Dockerfile

```diff
@@ -4,7 +4,7 @@ MAINTAINER Luis Fernando Gomes <your@luiscoms.com.br>
 
 ENV ERLANG_SOLUTIONS_VERSION 1.0-1
 RUN yum update -y && yum clean all
-RUN yum install -y wget && yum clean all
+RUN yum install -y wget epel-release && yum clean all
```

Comment:
Which version will be installed on this repository?

Reply:
I am not sure. I was not able to get the build to work without installing epel-release. You might be able to get yours to work without that specific change. The most relevant change is commit 41624fd36b3dd411a45a2447bf5ceb3b230697f2